src/erlcloud_elb.erl

```diff
@@ -211,6 +222,21 @@ describe_load_balancers(Names, Params, Config) ->
             {error, Reason}
     end.
 
+-spec describe_tags(ElbNames, Params, AwsConfig) -> paged_result()
```

Comment:
It would be nice to also have a arity 2 version omitting Params if it is possible to have a default values for Params.
Also it would be great to get describe_tags/3 that will get ElbNames and Marker to get next page, not whore Params array.

Reply:
Optionally you also can make describe_tags_all function that will return whole set of tags without paged results similar to how describe_load_balancers_all implemented below.